src/Pusher.php

```diff
@@ -55,6 +55,9 @@ class Pusher implements LoggerAwareInterface
      *                         useTLS - quick option to use scheme of https and port 443.
      *                         encrypted - deprecated; renamed to `useTLS`.
      *                         cluster - cluster name to connect to.
+     *                         encryption_master_key - the encryption master key used to encrypt events' data for private-encrypted channels
```

Comment:
What about: "This key, along with the channel name, are used to derive per-channel encryption keys. Per-channel keys are used encrypt event data on encrypted channels."

Since it's not the encryption_master_key which directly encrypts events.